library_generation/test/resources/integration/pr-description-golden.txt

```diff
@@ -0,0 +1,10 @@
+7a9a855287b5042410c93e5a510f40efd4ce6cb1
```

Comment:
I know we don't have the logic for telling exactly which library is affected, but maybe in the meantime we can format this commit hash to be an actual link to the commit in `googleapis` for further inspection. WDYT?

Reply:
I think this is a good idea, but we don't want to include them in the release notes, so please put it outside of `BEGIN_COMMIT_OVERRIDE` and `END_COMMIT_OVERRIDE` if you want to include them in the PR description.

Reply:
Done. The commit link is outside of `BEGIN_COMMIT_OVERRIDE` and `END_COMMIT_OVERRIDE`.